### DISCUSSION TITLE
High negative pressure and overlapping atoms from trained potential

### DISCUSSION THREAD

**Discussion by mayankaditya** [deepmodeling/deepmd-kit#228] [Q&A]
Hello Admin,

I am using the deepmd code to train the potential for one of my systems which is a disorder. I created the pair potential (using ~500000 batch, force error ~10meV/A). The generated lammps potential gives very high negative value of pressure and after few steps the diffusive atoms are overlapping/merge with each other. I train the model with the following parameters. 

**{
    "_comment": " model parameters",
    "use_smooth":        true,
    "sel_a":             [150,200,200],
    "rcut_smth":          2.0,
    "rcut":              10.00,
    "filter_neuron":     [25, 50, 100],
    "fitting_neuron":   [240,240,240],
    "filter_resnet_dt":  false,
    "axis_neuron":       32,
    "fitting_resnet_dt": true,
    "_comment":          "that's all",
    "_comment": " training controls",
    "systems":      ["./"],
    "set_prefix":   "set",
    "stop_batch":   1000000,
    "batch_size":   1,
    "start_lr": 0.001,
    "decay_steps":  5000,
    "decay_rate":   0.95,
    "start_pref_e": 0.02,
    "limit_pref_e": 8,
    "start_pref_f": 1000,
    "limit_pref_f": 1,
    "start_pref_v": 0,
    "limit_pref_v": 0,
    "seed":     1,
    "_comment": " display and restart",
    "_comment": " frequencies counted in batch",
    "disp_file":    "/dscrhome/mg422/lcurve.out",
    "disp_freq":    100,
    "numb_test":    100,
    "save_freq":    100,
    "save_ckpt":    "model.ckpt",
    "load_ckpt":    "model.ckpt",
    "disp_training":    true,
    "time_training":    true,
    "_comment": "that's all"
}

Kindly advise, how to deal with this issue.

Thanks,
Mayank

**Comment by amcadmus** [comment]
Hi Mayank,

Do you have the virial label in your data? If so could you please check the prediction accuracy on virial? And check if DFT gives you a high negative pressure. 

Best, Han
